render/mapping.go

```diff
@@ -276,7 +307,10 @@ func MapContainer2ContainerImage(n RenderableNode) (RenderableNode, bool) {
 		return n, false
 	}
 
-	return newDerivedNode(id, n), true
+	// Add container-<id> key to NMD, which will later be counted to produce the minor label
+	result := newDerivedNode(id, n)
+	result.NodeMetadata.Metadata[containersPrefix+n.ID] = ""
+	return result, true
```

Comment:
I'm not keen on special casing keys in the ndm metadata struct like that.  Thats a yellow flag to me.

As a half-way step between a counters field on renderable node, how about any keys in NMD.metadata with the prefix "-counter" gets add semantics?